extensions/ql-vscode/src/codeql-cli/distribution.ts

```diff
@@ -560,7 +601,8 @@ export class ReleasesApiConsumer {
   }
 
   public async getLatestRelease(
-    versionRange: semver.Range,
+    versionRange: semver.Range | undefined,
+    orderBySemver = true,
```

Comment:
Since you added this parameter here, there seem to be some call sites that are unintentionally setting `orderBySemver` instead of `includePrerelease` in the tests.

Reply:
Oops. Sorry I thought I got them all.

I wish I knew why "find referencs" often just ignores references in tests.